RELEASE.md

```diff
@@ -5,6 +5,24 @@ Once the code has been merged into the `develop` branch on this repo, there are
 - You should create a GitHub [tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging), with the appropriate version number. Typically, from `v21.06` onwards all tags are created following the Linux Ubuntu versioning convention which is the `YY.MM` format where `Y` is the year and `M` is the month of that year when that release was created.
 - You should push the package to [PyPI](https://pypi.org/). Schematic is on PyPI as [schematicpy](https://pypi.org/project/schematicpy/). You can go through the following two sections for that.
 
+## Steps 
+- Step 1: Delete the old `main` branch
+You could delete the current `main` branch locally by doing: `git branch -d main`. Then, to delete the `main` branch remotely, you could do: `git push origin --delete main`
```

Comment:
Remove the  delete on `main` as a standard process. In the future, the `main` branch will be closer to `develop` branch

Reply:
also add this: "in the release PR link all PRs and/or issues that are included in the release" to the documentation.